### PR TITLE
Added types for SIP.js

### PR DESCRIPTION
**Pull request by decyrus**
- [x] Make your PR against the `master` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).
- [x] Run `tsc` without errors.
- [x] Run `npm run lint package-name` if a `tslint.json` is present.

- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `npm run new-package package-name`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

### CODE REVIEW

File: sip.js/tslint.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "../tslint.json",
+    "rules": {
+        "no-empty-interface": false,
+        "export-just-namespace": false
```
Comment (mhegazy) [inline sip.js/tslint.json]: you do not need `export = SIP; declare namespace SIP`.

**Comment by pravdinalex** [comment]
@decyrus , thank you very much! I searched this wrapper past week and now it's here =)

File: sip.js/tslint.json (hunk 1 above)
Comment (decyrus) [inline sip.js/tslint.json]: Thank you for review! I fixed it.

**Comment by decyrus** [comment]
@mhegazy, could you review again and close this PR?

**Comment by DanielRosenwasser** [comment]
Thanks! Sorry for the delay!

**Pull request merged**